example/generator/SinewaveLimitProvider.js

```diff
@@ -42,13 +54,17 @@ define([
                 name: "Red High"
             },
             rl: {
-                cssClass: "is-limit--lwr is-limit--red",
+                color: 'red',
+                isUpper: false,
```

Comment:
isUpper seems to be being used inconsistently. Some limit evaluation results have it, some don't.

Reply:
Also, if `isUpper` is now  a required part of the limits API, it should be documented somewhere.

Reply:
@charlesh88 This was experimental to see if we could retain backwards compatibility. It look like the following classes are applied to the highlighted values that are out of limits but the classes aren't 'working'. Can you take a look?
`plot-series-value hover-value-enabled cursor-hover value-to-display-nearestValue  c-plot-limit--red c-plot-limit--lwr c-plot-limit--critical`

Reply:
@shefalijoshi I will look at this after Tuesday 6/21

Reply:
@akhenry I'm going to backout these changes for now. We'll create a different PR to make the old limits highlights work with the new limits work.